CMakeLists.txt

```diff
@@ -140,8 +138,8 @@ IF(UNIX)
   SET_TARGET_PROPERTIES(${PROJECT_NAME} PROPERTIES LINKER_LANGUAGE CXX)
   TARGET_INCLUDE_DIRECTORIES(${PROJECT_NAME} PUBLIC $<INSTALL_INTERFACE:include>)
   TARGET_LINK_LIBRARIES(${PROJECT_NAME} pinocchio::pinocchio)
-  TARGET_LINK_LIBRARIES(${PROJECT_NAME} ${Boost_FILESYSTEM_LIBRARY} ${Boost_SYSTEM_LIBRARY}
-      ${Boost_SERIALIZATION_LIBRARY})
+  TARGET_LINK_LIBRARIES(${PROJECT_NAME} Boost::filesystem Boost::system
```

Comment:
might not work for all the OS.

Reply:
For now, I'm just checking if this fix an issue for @edantec, but thanks for the heads-up, I'll add that if this works for him.

Reply:
done

Reply:
Again, this might not work for all the versions of Boost.

Reply:
I am not aware of any supported OS on which this wouldn't work. But if this can reassure you to make this a bit cleaner, I can find the minimal version of Boost on which this work, and set that as the minimal required Boost version.

Reply:
I think OS X won't work for instance and if I where, I would not change it, because there is no real need.

Reply:
I found the issue on @edantec's computer, it was unrelated to CMake. I'll close this.

Reply:
Thanks for letting us know.